app/models/manageiq/providers/lenovo/provider.rb

```diff
@@ -1,9 +1,10 @@
+# rubocop:disable Rails/HasManyOrHasOneDependent,Rails/InverseOf
```

Comment:
This should be ignored around the specific relationship, not for the whole file.

Reply:
Right.